### ISSUE TITLE
About train witn Raindrop dataset

### ISSUE THREAD

**Issue by liangVC** [Easquel/WaveDM#11]
When training with the Raindrop dataset, are the training parameters the same as those in raindrop_wavelet.yml? For example, is the batch size 1? Is the learning rate 0.00004? Could you please tell me all the actual parameters used? Thank you very much.

**Comment by Easquel** [comment]
The batch size can be adjusted according to your GPU memory with the learning rate being constant simultaneously. The learning process will continue for a long time. So actually there is no true ending point. You can just stop it if the performance satisfies you.
